README.md

```diff
@@ -508,46 +434,56 @@ $ make toolchains
 * `repo sync` can take an additional parameter -j to sync multiple remotes. For
    example `repo sync -j3` will sync three remotes in parallel.
 
-### 6.3. QEMU
-After getting the source and toolchain, just run:
+---
+### 5.3. QEMU
+After getting the source and toolchain, just run (from the `build` folder)
 ```
 $ make all run
 ```
 and everything should compile and at the end QEMU should start.
 
-### 6.4. FVP
-After getting the source and toolchain you must also get the get Foundation
-Model
+---
+### 5.4. FVP
+After getting the source and toolchain you must also obtain Foundation Model
 ([link](http://www.arm.com/products/tools/models/fast-models/foundation-model.php))
-and untar it to the forest root, then just run:
+binaries and untar it to the forest root, then just run (from the `build` folder)
+
 ```
 $ make all run
 ```
 and everything should compile and at the end FVP should start.
 
-### 6.5. Hikey
-After running `make` above, follow the instructions at
-[flash-binaries-to-emmc](https://github.com/96boards/documentation/wiki/HiKeyUEFI#flash-binaries-to-emmc-)
-to flash all the required images to and boot the board.
+---
+### 5.5. HiKey
+After getting the source and toolchain, just run (from the `build` folder)
```

Comment:
The Ubuntu package name may be `python-crypto`.
```
$ grep Signature scripts/sign.py 
	from Crypto.Signature import PKCS1_v1_5
$ ls /usr/lib/python2.7/dist-packages/Crypto/Signature
__init__.py  __init__.pyc  PKCS1_PSS.py  PKCS1_PSS.pyc  PKCS1_v1_5.py  PKCS1_v1_5.pyc
$ dpkg -S /usr/lib/python2.7/dist-packages/Crypto/Signature
python-crypto: /usr/lib/python2.7/dist-packages/Crypto/Signature
```

IIRC you need a recent version of the package otherwise the `Signature` module is not present -- see this line in `.travis.yml`: https://github.com/OP-TEE/optee_os/blob/master/.travis.yml#L50-L51 (in the Travis container environment `apt-get` cannot be used, that's why I had to use `pip`).